client/src/app/components/employee-settings/employee-settings.component.html

```diff
@@ -13,25 +13,20 @@
 
           <mat-form-field>
             <mat-label for="First Name"></mat-label>
-            <input matInput id="first_name" type="text" [disabled]="true" [value]="employee.first_name"/>
+            <input matInput id="first_name" type="text" [disabled]="!isLoggedInUser"  [value]="employee.first_name"/>
           </mat-form-field>
 
           <mat-form-field>
             <mat-label for="Last Name"></mat-label>
-            <input matInput id="last_name" type="text" [disabled]="true" [value]="employee.last_name"/>
+            <input matInput id="last_name" type="text" [disabled]="!isLoggedInUser" [value]="employee.last_name"/>
           </mat-form-field>
 
           <mat-form-field>
             <mat-label for="email">Email</mat-label>
             <input matInput id="email" type="text" [(ngModel)]="employee.email" email [disabled]="!edit"/>
           </mat-form-field>
 
-          <mat-form-field>
-            <mat-label for="password">Password</mat-label>
-            <input matInput id="password" type="password" [(ngModel)]="password" minlength="5" [disabled]="!edit"/>
-          </mat-form-field>
-
-          <mat-form-field>
```

Comment:
there's added functionality for adding a password, so why are we removing it from the form?

Reply:
No, because the update password uses a different endpoint. So it needs to be submitted in a different form. Also, later we might add a field another field to confirm the password. I feel its way better this way.

Reply:
oooh, I didn't notice that you added a new form